AnimalMatcher/AnimalMatcher.Data/AnimalMatcherDbContext.cs

```diff
@@ -37,6 +39,20 @@ protected override void OnModelCreating(ModelBuilder builder)
                 .HasForeignKey(like => like.LikedPetId)
                 .OnDelete(DeleteBehavior.Restrict);
 
+            builder
+                .Entity<Pet>()
+                .HasMany(pet => pet.SentMessages)
+                .WithOne(message => message.Sender)
+                .HasForeignKey(message => message.RecipientId)
```

Comment:
I think the RecipientId should be a foreign key for the Recepient and should be linked to ReceivedMessages in Pet

Reply:
Good point, updated with last commit.